components/admin/GlobalTransactionExplorer.jsx

```
@@ -0,0 +1,788 @@
+                  id="dateTo"
+                  type="date"
+                  value={filters.dateTo}
+                  onChange={(e) => handleFilterChange("dateTo", e.target.value)}
+                  className="h-9 text-sm"
+                  aria-label="Filter by end date"
+                />
+              </div>
+
+              {/* Status Filter */}
+              <div className="space-y-1.5">
+                <label className="text-xs font-medium text-muted-foreground">
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)